docs/compiling/compiling-on-wsl-ubuntu-24.04.md

```
@@ -0,0 +1,56 @@
+.
+├── remeres-map-editor
+└── vcpkg
+```
+
+## 5. Configure and build
+
+```bash
+cmake --preset linux-release -DTOGGLE_BIN_FOLDER=ON
+cmake --build --preset linux-release -j4
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)